src/components/data/dataFields.js

```diff
@@ -1,7 +1,9 @@
+import { DEFAULT_PAGE_SETTINGS } from "./utils";
+
 const dataFields = (dataI18n) => ({
     NAME: {
         fieldName: dataI18n("name"),
-        key: "name",
+        key: DEFAULT_PAGE_SETTINGS.orderBy,
```

Comment:
This feels weird to me since it's a key-value thing that's under a hardcoded `NAME` -- I guess I'd expect this to stay hardcoded, not use the default page settings map.

Reply:
Looking at this now, I agree - I'll switch it back